### DISCUSSION TITLE
Can py-pde solve Coupled nonlinear PDE ?

### DISCUSSION THREAD

**Discussion by rfatihors** [zwicker-group/py-pde#521] [Q&A]
<img width="504" alt="Ekran Resmi 2024-01-17 17 16 48" src="https://github.com/zwicker-group/py-pde/assets/108195555/224d1a72-e069-4339-8d4f-2a378185e187">

**Comment by david-zwicker** [comment]
Yes

**Reply by rfatihors** [reply]
How? I could not use it according to your examples. Please can you explain how to implement it? thanks in advance

**Reply by david-zwicker** [reply]
I don't have the bandwidth to implement concrete problems posted in these discussions. Please show what you've done yourself and where you cannot proceed. Clearly, reading [the documentation](http://py-pde.readthedocs.io) should help you.
